### PR TITLE
[WIP] djuran, imiller0 and yliu127 comments on consolidated branch

### PR DESCRIPTION
**Pull request by aireilly**
Ready to merge.

### CODE REVIEW

File: scalability_and_performance/ztp-deploying-disconnected.adoc [hunk 1]
```diff
@@ -73,7 +78,7 @@ include::modules/ztp-pgt-config-best-practices.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* For details about best practice for scaling clusters with {rh-rhacm-first}, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/install/installing#performance-and-scalability[ACM performance and scalability considerations].
+* For details about best practice for scaling clusters with {rh-rhacm-first}, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/adoc/install/installing#performance-and-scalability[ACM performance and scalability considerations].
```
Comment (amolnar-rh) [inline scalability_and_performance/ztp-deploying-disconnected.adoc:81]: @aireilly I don't think these links with `adoc` in them will work. Have you tested it in a local build?

File: scalability_and_performance/ztp-deploying-disconnected.adoc [hunk 2]
```diff
@@ -146,7 +159,7 @@ include::modules/ztp-site-cleanup.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 * For information on removing a cluster, see
-https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/clusters/managing-your-clusters#remove-managed-cluster[Removing a cluster from management]
+https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/adoc/clusters/managing-your-clusters#remove-managed-cluster[Removing a cluster from management]
```
Comment (amolnar-rh) [inline scalability_and_performance/ztp-deploying-disconnected.adoc:162]: Same as above. Plus `link:` is missing.

**Review by amolnar-rh — CHANGES_REQUESTED** [review]
Few comments only, please check them and I'll merge once you're finished.

File: scalability_and_performance/ztp-deploying-disconnected.adoc (hunk 1 above)
Comment (aireilly) [reply scalability_and_performance/ztp-deploying-disconnected.adoc:81]: argh. Good catch. will fix.

**Pull request merged**